sonic-xcvrd/scripts/xcvrd

```diff
@@ -389,6 +389,33 @@ def waiting_time_compensation_with_sleep(time_start, time_to_wait):
     if time_diff < time_to_wait:
         time.sleep(time_to_wait - time_diff)
 
+def wait_for_all_ports_configured():
+    # Connect to APPL_DB and subscribe to PORT table notifications
```

Comment:
This table is not intended to be subscribed outside of orchagent.

Reply:
This subscription is NOT recent changes. Its existing since 201811.

Reply:
This can potentially break orchagent functionality. We need to revisit this and not add more case.

Reply:
Unblocking as this is already existing code. Let's eventually move this subscription to STATE_DB instead.